src/fulltrade/FullTradeCard.js

```diff
@@ -51,6 +51,7 @@ export default class GenericTradeCard extends Component {
         trade: PropTypes.object.isRequired,
         type: PropTypes.oneOf(['tick', 'full']).isRequired,
         tick: PropTypes.array,
+        isAuthorized: PropTypes.bool.isRequired,
```

Comment:
May be we want to abstract this more like 'isAvailableForTrade'.
Need to discuss it three of us later.